doc/develop/west/zephyr-cmds.rst

```diff
@@ -281,3 +281,123 @@ Additional tips:
   separate ``west global`` command since ``global`` already searches for the
   ``GTAGS`` file starting from your current working directory. This is why you
   need to run ``global`` from inside the workspace.
+
+.. _west-patch:
+
+Working with patches: ``west patch``
+************************************
+
+The ``patch`` command allows users to apply patches to Zephyr or Zephyr modules
+in a controlled manner that makes automation and tracking easier for external applications that
+use the :ref:`T2 star topology <west-t2>`. The :ref:`patches.yml <patches-yml>` file stores
+metadata about patch files and fills-in the gaps between official Zephyr releases, so that users
+can easily see the status of any upstreaming efforts, and maintainers can easily determine which
+patches to drop before upgrading to the next Zephyr release.
```

Comment:
I don't understand the "and maintainers can easily determine which patches to drop before upgrading to the next Zephyr release" part?

Reply:
yeah it's the use of "maintainers" in that context which is confusing. In Zephyr documentation, "maintainer" really ought to mean "Zephyr maintainer"

So I think you want to say the below, no?

> so that users can easily see the status of any upstreaming efforts, and determine which
patches to drop before upgrading to the next Zephyr release.

Reply:
I see - sure I can change that.

Reply:
Thanks!